crates/router/src/core/payment_methods/transformers.rs

```diff
@@ -985,6 +985,14 @@ impl transformers::ForeignTryFrom<domain::PaymentMethod> for api::CustomerPaymen
         // TODO: check how we can get this field
         let recurring_enabled = true;
 
+        let psp_tokenization_enabled = item.connector_mandate_details.and_then(|details| {
+            details.payments.map(|payments| {
+                payments.0.iter().any(|(_, value)| {
+                    value.connector_token_status == api_enums::ConnectorTokenStatus::Active
+                })
+            })
```

Comment:
Nit: Can use `values()` instead of `iter()`, if we're not accessing the keys here.

```suggestion
            details.payments.map(|payments| {
                payments.0.values().any(|connector_token_reference| {
                    connector_token_reference.connector_token_status == api_enums::ConnectorTokenStatus::Active
                })
            })
```

Also, should we consider implementing `Deref` on `PaymentsTokenReference` (to avoid the `.0` field access) @Narayanbhat166?